proto/desmos/profiles/v1beta1/msgs.proto

```diff
@@ -46,6 +47,17 @@ service Msg {
 
   // UnblockUser defines a method for unblocking a user
   rpc UnblockUser(MsgUnblockUser) returns (UnblockUserResponse);
+
+  // CreateIBCAccountLink defines a method for creating an account link to other
+  // chain
+  rpc CreateIBCAccountLink(MsgCreateIBCAccountLink)
+      returns (MsgCreateIBCAccountLinkResponse);
+
+  // Link defines a method for linking an account
+  rpc Link(MsgLink) returns (LinkResponse);
+
+  // Unlink defines a method for unlink an account from a profile
+  rpc Unlink(MsgUnlink) returns (UnlinkResponse);
```

Comment:
I think it's better to rename the methods, messages and responses to be: 

- `LinkAccount`, `MsgLinkAccount`, `LinkAccountResponse`
- `UnlinkAccount`, `MsgUnlinkAccount` and `UnlinkAccountResponse` 

This is just to make it easier we are linking accounts and not other things (eg. third-party applications)